Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionViewController.swift

```diff
@@ -400,7 +400,7 @@ class ReviewExecutionViewController: ContainerViewController, PasscodeProtecting
     }
 
     func didLoadPaymentData() {
-        if controller.relaysRemaining > ReviewExecutionViewController.MIN_RELAY_TXS_LEFT && !self.userSelectedSigner {
+        if controller.relaysRemaining > ReviewExecutionViewController.MIN_RELAY_TXS_LEFT && !self.userSelectedSigner && safe.chain!.isSupport(feature: .relay) {
```

Comment:
not related to this pr, but what is the point of having MIN_RELAY_TX_LEFT constant that is zero?

Reply:
For debugging. You can set this to 4 and you only need to relay on tx to check the other flow. If it is not a constant you have to change zeroes in several places to achieve the same effect.

Reply:
I'm also not in favor of pushing debug code, but this has been done in different pr

Reply:
This is not debugging code. It makes sense to have this exatracted into one constant. Instead of usinf "0" in several places.